gpflow/models/sgpr.py

```diff
@@ -211,6 +211,31 @@ def predict_f(self, X: tf.Tensor, full_cov=False,
             var = tf.tile(var[:, None], [1, self.num_latent])
         return mean + self.mean_function(X), var
 
+    def compute_qu(self):
+        """
+        Computes the mean and variance of q(u), the variational distribution on
+        inducing outputs. SVGP with this q(u) should predict identically to
+        SGPR.
+        :return: mu, A
```

Comment:
What is A? the covariance of q(u)? or its cholesky? would be good if this could be consistent in code & docstring, e.g. "q(u) = N(mean, cov)" and `:return: mean, cov` or `:return mean, cholesky(cov)` ...

Reply:
or `q_mu` and `q_sqrt` if it is the cholesky